src/commands/moderation.rs

```diff
@@ -427,8 +440,14 @@ pub async fn purge(
                 serenity::CreateEmbed::default()
                     .title("Purge successful")
                     .field("Deleted messages", deleted_amount.to_string(), false)
+                    .field("Action by", author.mention(), false)
                     .color(embed_color)
-                    .thumbnail(image)
+                    .thumbnail(image.clone())
```

Comment:
```suggestion
                    .thumbnail(&image)
```
Realized it is better to borrow here sorry